crates/holochain/src/core/workflow/incoming_dht_ops_workflow/tests.rs

```diff
@@ -208,9 +208,9 @@ async fn get_pending_op_hashes(env: DbWrite<DbKindDht>) -> Vec<DhtOpHash> {
 }
 
 async fn clear_requires_receipt(env: DbWrite<DbKindDht>, op_hashes: Vec<DhtOpHash>) {
-    env.read_async(move |mut txn| -> StateMutationResult<()> {
+    env.write_async(move |txn| -> StateMutationResult<()> {
```

Comment:
What are the differences and use-cases for `read_async` vs `write_async`? I would guess that `read_async` is only for read operations but then why did it need `mut` with read but not write?